### ISSUE TITLE
Paladin testing Jan 2022

### ISSUE THREAD

**Issue by warius1983** [datchannin/Brothers-WoW#64]
After playing my last adventure I tried to find some new spots for the paladin and tried something different, as a support tank it worked enough well, but still something is missing to him.

I have been analyzing the logs after each battle and I found out 2 possible ways to make it more unique.

Aoe damage: Paladin will never beat a warrior on DPS both in ST and AOE, but can be much more efficent in fatique and for AP spent, Divine storm could be the answer and i found, and exoprted and tested the correct lines from legends:

```
	_function onAnySkillUsed( _skill, _targetEntity, _properties )
	{
		if (_skill.isAOE() && _skill.isAttack())
		{
			_properties.DamageTotalMult *= 1.2;
		}
	}_
```

It works perfectly with any spell and any weapon

It's 20% it could stay both in T4 or T6 row perk.

The other spell I found a very good to add is avenging wrath, cooldown DPS, in wow you deal extra damage and spells cost less, always in legends i found these files.

[perfect focus.zip](https://github.com/datchannin/Brothers-WoW/files/7830779/perfect.focus.zip)

Essentially this spell is a use cast that when used reduces the ap cost of all action by 50%, but increases all fatique cost by 100%, more or less, It could be nice that aveging wrath had a cooldown I don't know 3-4 rounds, costs like 100 fatique and 0 AP.
While you have the buff, your actions cost half AP but 0 Fatique, this way after the buff turns off ( only 1 turn ) you have to refill his fatique bar or by breath or by druid, do you like the idea?

**Comment by datchannin** [comment]
I agree that paladin is not so good damage dealer as cat/warrior/shaman/rogue
and paladin is not so good at healing as priest/shaman/druid (because he can not remove injuries or repair armor)

I found several places in roster for paladin. Mostly now I use him for stun powerful enemies (as fallen paladin or fel/frost mages). He is only one now who can stun with 100% probability and in middle range. After paladin learns Seal of Command he can use two-handed hit (6 AP) + stun one enemy (4 AP). Or he can use 2 stuns per one turn. Now I nerfed power of new enemies, but paladin still good in control.

Second place for paladin I want is healer with remove all status debuff (bleed, poison, stun, daze). But seems 2 perks for heal + 3 perks for clean status + 2 perks for aura + 2 perks for buffs are too expensive only for average healer. Priests and Shamans seems much better. May be all 'clean' perks should be collected to one perk. And may be additional power for remove injuries should be added. And then paladin will be able heal instead of Holy priest (disc+disc+pal setup, or disc+sham+pal setup). 

I agree, may be paladin can be empower for use some AOE spells. I plan to return to paladin for update some his perks and may be try new features.
